vrk-kpa/suomidigi
# SUOPA-857 Hyphens

To test: 
- `make fresh drush-cr`
- Go to https://suomidigi.docker.sh/ohjeet-ja-tuki/menetelmat/kokonaisarkkitehtuurimenetelma and resize the browser until the heading is getting too long for the reserved content area. Verify that the heading will get hyphened correctly.
- Also test on other than macOS or Android device; These devices support hyphens:auto - see: https://caniuse.com/#search=CSS%20property%3A%20hyphens
